website/app/styles/doc-components/color-swatch.scss

```
@@ -0,0 +1,81 @@
+$doc-color-swatch-border-radius: 3px;
+
+.doc-color-swatch {
+  display: grid;
+  grid-template-rows: 64px 1fr;
+  width: 100%;
+  margin: 16px 0;
+  padding: 0;
+  border: 1px solid var(--doc-color-gray-500);
+  border-radius: $doc-color-swatch-border-radius;
+
+
+  @media screen and (min-width: 1280px) {
+    grid-template-rows: 1fr;
+    grid-template-columns: (minmax(80px, 80px) auto); // just 80px auto didn't work, maybe a bug?
+  }
+}
+
+.doc-color-swatch__preview {
+  border-top-left-radius: $doc-color-swatch-border-radius;
+  border-top-right-radius: $doc-color-swatch-border-radius;
+
+  @media screen and (min-width: 1280px) {
+    grid-area: 1 / 1 / 2 / 2;
+    max-width: 80px;
+    border-top-right-radius: 0;
+    border-bottom-left-radius: $doc-color-swatch-border-radius;
+  }
+}
+
+.doc-color-swatch__info {
+  width: 100%;
+  padding: 16px;
+}
+
+.doc-color-swatch__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.doc-color-swatch__name {
+  margin: 0 0 8px 0;
+  font-weight: bold;
+  font-weight: 700;
+  font-size: 1.125rem;
+  font-family: Gilmer, sans-serif;
```

We could probably use the mixin here to avoid having to define the whole stack here 
```suggestion
  @include doc-font-family("gilmer");
```

requires
```scss
@use "../typography/mixins" as *;
```

---

ok I have added this for now, I'm going to schedule some time to chat because I don't think I really understand how these mixins are setup.

---

I'm not super familiar with them myself, but they seem like a decent way to apply typography styling consistently 😁